internal/project/teststub/teststub.go

```diff
@@ -15,6 +15,7 @@ func Setup(t *testing.T, temp string) Path {
 
 	testDataSrc := originalTestDataPath()
 	require.NoError(t, copy.Copy(testDataSrc, temp))
+	Cleanup(t, temp)
```

Comment:
@pastuxso did you encounter any problem that led to creating this helper? Overall, when using `T.TempDir()` it is cleared automatically when the test case finishes.

Reply:
@adambabik When I run the tests on Linux, it's failing, even individually performed.

This is an example:

![image](https://github.com/stateful/runme/assets/43882/ef7dfb39-808f-4a56-a370-0ba5b9172bdd)